rasa_core/policies/fallback.py

```diff
@@ -93,7 +93,11 @@ def predict_action_probabilities(self,
         # if NLU interpreter does not provide confidence score,
         # it is set to 1.0 here in order
         # to not override standard behaviour
-        nlu_confidence = nlu_data["intent"].get("confidence", 1.0)
+        if nlu_data["intent"]:
+            nlu_confidence = nlu_data["intent"].get("confidence", 1.0)
+        else:
+            logger.warning("NLU intend is None.")
```

Comment:
what do we need this logging message for?

Reply:
I wasn't sure if no intent is an issue that should be fixed. Since it seems to be normal for some policies should I remove this or make it a debug logging?

Reply:
yes, I think we can remove it :+1: